### DISCUSSION TITLE
Welcome to webview_cef Discussions!

### DISCUSSION THREAD

**Discussion by hlwhl** [hlwhl/webview_cef#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by medobedo-glitch** [comment]
does the webview widget saves cache of a website?

**Reply by hlwhl** [reply]
Yes, you can use devTool to check how your content be cached.

**Comment by BullsEye34** [comment]
Hi @hlwhl In the [README.md](https://github.com/hlwhl/webview_cef/discussions/15#readme) of the project, under macOS section, there is a note which says the following.

> Note: Currently the project has not been enabled with multi process support due to debug convenience. If you want to enable multi process support, you may want to enable multi process mode by changing the implementation and build your own helper bundle. (Finding a more elegant way in the future.)

I was wondering if there are any instructions on how to do that. I want to have multi tab support in an application I am building, and right now, Every tab loads the same content even though different controllers are passed to the WebView widget. I do not possess knowledge of C or ObjC much, only the basics of Swift.

Is this the right path to take? Or is there any other way I can achieve the same?
Thank you so much.
